### ISSUE TITLE
How to do manual calculation in Neural Network

### ISSUE THREAD

**Issue by dragz17** [keras-team/keras#10285]
## Description:
I have an issue, i don't know how to do calculation for getting the accuracy of training, validation and testing.

## Ask
Can you help me?

**Comment by dragz17** [comment]
After doing flattening, the matrix change into (n,1) and the data became input layer.
Please help me, i dont know how to do it.

**Comment by JamesTrick** [comment]
Hey @dragz17 

Do you have any code that you've written already? It'll help me provide a contextual answer. In the meantime, here's some pseudocode below on how you might get the accuracy and loss of a model.


```python
# Define model, this is a simple 1 layer NN. This will be created by you.
model = models.Sequential()
model.add(Dense(16, activation='relu', input_shape=(x, )))
model.add(Dense(1, activation='sigmoid')

model.compile(optimizer=RMSprop, loss='binary_cross_entropy', metrics['binary_accuracy])

# Save the fit to a variable of your choice, to be accessed later.
history = model.fit(X_train, y_train, epochs=20, validation_split=0.2)

history_dict = history.history

# use history_dict .keys() to see what happened during training

# displays the training accuracy
history['acc']

# displays the validation accuracy
history_dict['val_acc']
```

Let me know if you have any other questions, or if I've misunderstood your question.

Cheers
James
